goon/code/datums/browserOutput.dm

```diff
@@ -59,10 +59,11 @@ var/list/chatResources = list(
 		for(var/asset in global.chatResources)
 			owner << browse_rsc(file(asset))
 
-		owner << browse(file("goon/browserassets/html/browserOutput.html"), "window=browseroutput")
-		sleep(20 SECONDS)
-		if(!owner || loaded)
-			break
+		for(var/subattempts = 1 to 3)
```

Comment:
the format we've decided to go with for loops in these is this:
`for(var/i in 1 to X)`

It's been demosntrated to not only be the fastest, but we've preferred the readability of it, as well.